### PR TITLE
[Docu] Quantity Price Rules

### PR DESCRIPTION
**Pull request by lukadschaak**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | -

Documenting the usage of quantity price rules.

With help by @Theux94

### CODE REVIEW

File: docs/02_User_Documentation/03_Price_Rules/03_Specific_Price_Rules.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# CoreShop Specific Price Rules
+
+Specific Prices are basically the same as Product Price Rules, but are configured directly on the
+product and therefore are only applied on that product.
+
+It is recommended to have at least one specific price, which is always applicable. For example a
```
Comment (dpfaffenbauer) [inline docs/02_User_Documentation/03_Price_Rules/03_Specific_Price_Rules.md:6]: it is not, why should it?

File: docs/02_User_Documentation/03_Price_Rules/04_Quantity_Price_Rules.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# CoreShop Quantity Price Rules
+
+Quantity Price Rules are configured directly on the product and therefore are only applied
+on that product. The settings are different to specific price rules. Calculation Behaviour
+means the property on which the amount is determined. This is volume per default.
+
+Quantity price rules are only applied to the price in context of a cart. In other cases the
+price calculator ignores them. So it is recommended to not have quantity price rules alone,
+but to have at least a default specific price (or a store values entry).
```
Comment (dpfaffenbauer) [inline docs/02_User_Documentation/03_Price_Rules/04_Quantity_Price_Rules.md:9]: Don't recommend things here.

File: docs/03_Development/03_Products/02_Price_Calculation.md [hunk 3]
```diff
@@ -19,6 +19,8 @@ CoreShop Price Calculation consists of 3 different prices:
  - **Discount**: Discount from promotions
  - **Price**: Retail Price or Discount Price (if available) minus discount rules
 
+To get a non-zero price, the calculation needs at least a matching specific price rule, or a store value to work.
```
Comment (dpfaffenbauer) [inline docs/03_Development/03_Products/02_Price_Calculation.md:22]: please remove. the default is ALWAYS to have a store-value. price

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]

**Comment by lukadschaak** [comment]
Updated, please check changes.

**Comment by lukadschaak** [comment]
*push* @dpfaffenbauer Still interested in this part of documentation?

**Comment by dpfaffenbauer** [comment]
@lukadschaak yes, sorry, I am on holiday right now and had lots todo before I left. Will review this when I am back.

**Comment by lukadschaak** [comment]
Ohh holiday is nice! Keep on refreshing!! :)

**Review by dpfaffenbauer — APPROVED** [review]

**Comment by dpfaffenbauer** [comment]
thanks a lot

**Pull request merged**